apps/prairielearn/src/ee/pages/administratorInstitutionSso/administratorInstitutionSso.tsx

```diff
@@ -57,13 +64,37 @@ router.get(
       req.params.institution_id,
     );
 
+    const pageContext = extractPageContext(res.locals, {
+      pageType: 'plain',
+      accessType: 'instructor',
+      withAuthzData: false,
+    });
+
     res.send(
-      AdministratorInstitutionSso({
-        supportedAuthenticationProviders,
-        institution,
-        institutionSamlProvider,
-        institutionAuthenticationProviders,
-        resLocals: res.locals,
+      PageLayout({
+        resLocals: { ...res.locals, institution },
```

Comment:
This feels weird, is that how the existing page worked?

Reply:
Yes. Look at the previous file on `master`:

https://github.com/PrairieLearn/PrairieLearn/blob/c6ad45326ffe3fe2e306e8bf4ed5ede1f1636818/apps/prairielearn/src/ee/pages/administratorInstitutionSso/administratorInstitutionSso.html.tsx#L33